app/Console/Commands/TeamSpeak/TeamSpeakDaemon.php

```diff
@@ -47,7 +47,13 @@ public function handle()
             try {
                 self::$connection->getAdapter()->wait();
             } catch (TeamSpeak3_Transport_Exception $e) {
-                self::$connection = $this->establishConnection();
+                try {
+                    self::$connection = $this->establishConnection();
+                } catch (TeamSpeak3_Transport_Exception $e) {
+                    // Connection failed, let the loop restart and try again
+                    $this->log('TeamSpeak connection failed: '.$e->getMessage(). '. Trying again in 15 seconds...');
```

Comment:
Are we potentially going to miss this if it isn't working for a few days? Do we need to try three times and then throw an exception or something?

Reply:
Do you mean something like I just pushed?